content/how-to/add-code-servers/gitlab-cloud/add-token/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "How To generate a personal access token (PAT) in GitLab"
+description: "How To generate a personal access token (PAT) in GitLab"
+lead: "How To generate a personal access token (PAT) in GitLab"
```

Comment:
Note that the `lead` is repeated under the title, so if we provide a lead, it should be different. I've been leaving in blank because of that.